src/Translator/assets/test/formatters/intl-formatter.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import {formatIntl} from '../../src/formatters/intl-formatter';
+
+describe('Intl Formatter', function () {
+    test('format with named arguments', function() {
+        const chooseMessage = `
+            {gender_of_host, select,
+                female {{num_guests, plural, offset:1
+            =0 {{host} does not give a party.}
+            =1 {{host} invites {guest} to her party.}
+            =2 {{host} invites {guest} and one other person to her party.}
+            other {{host} invites {guest} as one of the # people invited to her party.}}}
+        male   {{num_guests, plural, offset:1
+            =0 {{host} does not give a party.}
+            =1 {{host} invites {guest} to his party.}
+            =2 {{host} invites {guest} and one other person to his party.}
+            other {{host} invites {guest} as one of the # people invited to his party.}}}
+        other {{num_guests, plural, offset:1
+            =0 {{host} does not give a party.}
+            =1 {{host} invites {guest} to their party.}
+            =2 {{host} invites {guest} and one other person to their party.}
+            other {{host} invites {guest} as one of the # people invited to their party.}}}}
+        `.trim();
+
+        const message = formatIntl(chooseMessage, {
+            gender_of_host: 'male',
+            num_guests: 10,
+            host: 'Fabien',
+            guest: 'Guilherme',
+        }, 'messages+intl-icu', 'en');
+
+        expect(message).toEqual('Fabien invites Guilherme as one of the 9 people invited to his party.');
+    })
+
+    test('percents and brackets are trimmed', function() {
+        expect(formatIntl('Hello {name}', { name: 'Fab'}, 'messages+intl-icu', 'en')).toEqual('Hello Fab');
+        expect(formatIntl('Hello {name}', { '%name%': 'Fab'}, 'messages+intl-icu', 'en')).toEqual('Hello Fab');
+        expect(formatIntl('Hello {name}', { '{{ name }}': 'Fab'}, 'messages+intl-icu', 'en')).toEqual('Hello Fab');
+    });
+
+    test('format with HTML inside', function() {
+        expect(formatIntl('Hello <b>{name}</b>', { name: 'Fab'}, 'messages+intl-icu', 'en')).toEqual('Hello <b>Fab</b>');
+        expect(formatIntl('Hello {name}', { name: '<b>Fab</b>'}, 'messages+intl-icu', 'en')).toEqual('Hello <b>Fab</b>');
+    })
```

Comment:
We want to ensure that the issue fixed in https://github.com/willdurand/BazingaJsTranslationBundle/pull/325 is not present here.